toolchain/parser/parse_node_kind.h

```diff
@@ -57,7 +57,7 @@ class ParseNodeKind {
 
   // Enable conversion to our private enum, including in a `constexpr` context,
   // to enable usage in `switch` and `case`. The enum remains private and
-  // nothing else should be using this.
+  // nothing else should be using it.
```

Comment:
Here, "it" would seem to refer to the enum and that would be wrong. I think the original wording with "this" is more correct here.

Reply:
Ah I actually thought that it was trying to refer to the enum 🤔

Reply:
```suggestion
  // nothing else should be using this function.
```

If it helps, we can be explicit about the target.

Reply:
Looks good now.